### DISCUSSION TITLE
presents

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#1205] [2023 Attraction Contributions]
what do you get your loved ones as a gift?
Are flowers necessary?

**Comment by hernessteve** [comment]
I always order flowers for my mom and my sister.
We don't give each other expensive presents, but trying to spend more time with each other as mush as possible

**Comment by MirkosMangle** [comment]
My wife loves flowers, so I’m a frequent customer at [flying flowers](https://flying-flowers.pissedconsumer.com/customer-service.html). She doesn’t have any specific favorites, just enjoys a variety of mixed arrangements. It’s always nice to surprise her with something beautiful and fresh. If you have any recommendations for other floral services or special arrangements, I’d love to hear them
